tinygrad/mlops.py

```diff
@@ -55,6 +55,15 @@ def forward(self, x:LazyBuffer) -> LazyBuffer:
   def backward(self, grad_output:LazyBuffer) -> LazyBuffer:
     return self.ret.binary_op(BinaryOps.MUL, grad_output)
 
+class Sigmoid(Function):
+  __slots__ = "ret"
+  def forward(self, x:LazyBuffer) -> LazyBuffer:
+    self.ret = x.const_like(1).binary_op(BinaryOps.DIV, x.const_like(1).binary_op(BinaryOps.ADD, x.binary_op(BinaryOps.MUL, x.const_like(-math.log(math.e)/math.log(2))).unary_op(UnaryOps.EXP2)))
```

Comment:
`math.log(math.e) = 1`

You can replace this in the other places too, it's just dumb.